### ISSUE TITLE
fluid-demo.exe crashed

### ISSUE THREAD

**Issue by foskel** [lirios/fluid#262]
I have a Qt Creator project setup with Qbs. I have fluid and my qbs project in there. When I try to run the demo I get the following error:

```
Starting <path>/build-untitled-Desktop_Qt_5_11_0_MSVC2017_64bit-Debug\qtc_Desktop_Qt_5_11_0_MSVC2017_64bit_Debug\install-root\fluid-demo.exe...
The program has unexpectedly finished.
The process was ended forcefully.
<path>/build-untitled-Desktop_Qt_5_11_0_MSVC2017_64bit-Debug/qtc_Desktop_Qt_5_11_0_MSVC2017_64bit_Debug/install-root/fluid-demo.exe crashed.
```

I also tried to run it with the command prompt (including all DLLs) but it just closes as soon as it starts... 
Also, if I try to run the example (minimalqbs) I get this error:

```
qrc:/main.qml:1 plugin cannot be loaded for module "QtQuick": Cannot load library C:\Qt\Qt5.11.0\5.11.0\msvc2017_64\qml\QtQuick.2\qtquick2plugind.dll: The specified module could not be found.
```

My project looks like this:

![qtrc](https://user-images.githubusercontent.com/39450823/41671532-c40734b4-748d-11e8-902d-7a63b6c7bbd8.png)

PS: I had to change two lines of minimalqbs.qbs:

```
    // Include Fluid project
    SubProject {
        filePath: "fluid/fluid.qbs"

        Properties {
            documentationEnabled: false -> withDocumentation: false
            demoEnabled: true -> withDemo: true
            autotestEnabled: false
        }
    }
```

**Comment by foskel** [comment]
I fixed it by updating the qbs-shared modules. Now it starts up, stays blank and crashes...

**Comment by luntik2012** [comment]
This is duplicate of #257. I will make pull request after #261 approving (some branching issues), or you  can make it yourself.

**Comment by foskel** [comment]
The crash message of the demo is the same, with the difference that now it crashes after 20 seconds of the application being unresponsive.

**Comment by luntik2012** [comment]
could you show content of "C:\Qt\Qt5.11.0\5.11.0\msvc2017_64\qml\" and, if exists, "C:\Qt\Qt5.11.0\5.11.0\msvc2017_64\qml\QtQuick.2" directories?

**Comment by foskel** [comment]
I solved this by creating a new MSVC 2015 x86 project.
